src/plans/debug.jl

```diff
@@ -1364,7 +1364,7 @@ function DebugActionFactory(d::Symbol)
     (d == :Feasibility) && return DebugFeasibility()
     (d == :Stepsize) && return DebugStepsize()
     (d == :Stop) && return DebugStoppingCriterion()
-    (d == :WarnBacktracking) && return DebugWarnIfBacktrackingNotConverged()
+    (d == :WarnBacktracking) && return DebugWarnIfStepsizeCollapsed()
```

Comment:
```suggestion
    (d == :WarnStepsize) && return DebugWarnIfStepsizeCollapsed()
```

since we could use that for general step size rules, when they collapse, maybe also the symbol could reflect that? (Fix L 1344 then as well)